src/main/java/net/dries007/tfc/objects/blocks/stone/BlockRockRaw.java

```diff
@@ -31,8 +33,27 @@ public BlockRockRaw(Rock.Type type, Rock rock)
         super(type, rock);
     }
 
-    //todo: add collapsable mechanics
+    @Nullable
+    @Override
+    public BlockPos getFallablePos(World world, BlockPos pos)
+    {
+        return type.canFall() && shouldFall(world, pos) ? pos : null;
+    }
+
+    @SuppressWarnings("deprecation")
+    @Override
+    public void neighborChanged(IBlockState state, World worldIn, BlockPos pos, Block blockIn, BlockPos fromPos)
```

Comment:
This is different from 1.7.10 - the only thing that should trigger a collapse is a block being broken.